lib/streamlit/report_session.py

```diff
@@ -544,7 +544,7 @@ def _enqueue_script_request(self, request, data=None):
 
         """
         if self._state == ReportSessionState.SHUTDOWN_REQUESTED:
-            LOGGER.warning("Discarding %s request after shutdown" % request)
+            LOGGER.warning(f"Discarding {request} request after shutdown")
```

Comment:
Not your fault, but this line should be `LOGGER.warning("Discarding %s request after shutdown", request)` (it should not have used the format operator in the first place)